src/go/k8s/api/vectorized/v1alpha1/cluster_types.go

```diff
@@ -436,13 +439,15 @@ type ClusterCondition struct {
 }
 
 // ClusterConditionType is a valid value for ClusterCondition.Type
-// +kubebuilder:validation:Enum=ClusterConfigured
+// +kubebuilder:validation:Enum=ClusterConfigured;OperatorQuiescent
 type ClusterConditionType string
 
 // These are valid conditions of the cluster.
 const (
 	// ClusterConfiguredConditionType indicates whether the Redpanda cluster configuration is in sync with the desired one
 	ClusterConfiguredConditionType ClusterConditionType = "ClusterConfigured"
+	// OperatorQuiescentConditionType indicates that the controller has no outstanding work to do, based on the observedGeneration.
+	OperatorQuiescentConditionType ClusterConditionType = "OperatorQuiescent"
```

Comment:
@chrisseto  Using "OperatorQuiescent" instead of "ControllerQuiescent", so the term "controller" cannot get mixed up with redpanda's Controller Leader.